cms/static/js/features_jsx/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Thanks for the explanation, @arizzitano! I thought we were using airbnb in our global config set, but it looks like we're only using airbnb-base. Does that not have the same benefit?

https://github.com/edx/eslint-config-edx/blob/master/packages/eslint-config-edx/.eslintrc.json

I agree that we should support JSX in our shared ruleset.